src/v2/Components/Authentication/Views/SignUpForm.tsx

```diff
@@ -18,6 +18,7 @@ import { AuthenticationPasswordInput } from "../Components/AuthenticationPasswor
 import { AuthenticationCheckbox } from "../Components/AuthenticationCheckbox"
 import { AuthenticationFooter } from "../Components/AuthenticationFooter"
 import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { MntnConversionPixel } from "../../../System/Analytics/MNTN"
```

Comment:
ditto on abs path

Reply:
Also, since we'll probably not expand the Analytics folder as most of our tracking / analytics stuff is dynamically embedded via segment, it would be a bit simpler to move this to `v2/Components/MTNTrackingPixel.tsx` and treat it like a normal component.